src/tests/operate.test.js

```diff
@@ -0,0 +1,35 @@
+import operate from '../logic/operate';
+
+describe('Testing operate.js', () => {
+  test('Plus operation', () => {
+    expect(operate(10, 6, '+')).toBe('16');
+  });
+
+  test('Minus operation', () => {
+    expect(operate(8, 5, '-')).toBe('3');
+  });
+
+  test('Minus operation', () => {
+    expect(operate(10, 54, '-')).toBe('64');
+  });
+
+  test('Divide operation', () => {
+    expect(operate(8, 4, '÷')).toBe('2');
+  });
+
+  test('Divide operation', () => {
+    expect(operate(10, 5, '÷')).toBe('2');
+  });
+
+  test('Multy operation', () => {
+    expect(operate(2, 4, 'x')).toBe('8');
+  });
+
+  test('Multy operation', () => {
+    expect(operate(9, 9, 'x')).toBe('81');
+  });
+
+  test('Modulo operation', () => {
+    expect(operate(5, 4, '%')).toBe('1');
+  });
+});
```

Comment:
- Hey Team, good job with the test, however, some tests are failing, please, kindly check the Minus test to fix the issue. see the image below:

![Screenshot from 2022-12-08 17-16-15](https://user-images.githubusercontent.com/59397159/206503045-39fda34a-ce56-4ae2-9d40-9b333de2b644.png)


![Screenshot from 2022-12-08 17-14-24](https://user-images.githubusercontent.com/59397159/206503236-f09daac8-471a-40e8-8d07-cb0ba5c55b99.png)